cdap-docs/admin-manual/source/operations/preferences.rst

```diff
@@ -3,23 +3,38 @@
     :copyright: Copyright © 2014-2015 Cask Data, Inc.
 
 ============================================
-Runtime Arguments
+Preferences
 ============================================
 
 .. include:: ../../../_common/_include/include-v260-deprecate-procedures.rst
 
-Flows, Procedures, MapReduce programs, and Workflows can receive runtime arguments:
+Preferences provide the ability to save configuration at various levels of the system including CDAP Instance,
+Namespace, Application and Program. Configuration is represented by a map of string-string pairs. Preferences
+can be retrieved/saved/deleted through REST API and/or through CLI. When programs are started, the Preferences
+at different levels are collapsed into one map and it is accessible through ways described later. Preferences
+are persisted across restart of Programs.
 
-- For Flows and Procedures, runtime arguments are available to the ``initialize`` method in the context.
+The overriding order of Preferences is:
```

Comment:
The order of precedence of Preferences (with the highest on the left and lowest on the right) is:

    Runtime Arguments > Program > Application > Namespace > CDAP Instance

Runtime arguments are passed in during the start of the Program.

\[Comments:]
**How do runtime arguments work now vis-a-vis the saving of them for the lifecycle? Are they no longer persisted, but replaced by preferences?** 

See [Lifecycle RESTful API](http://stg-web101.sw.joyent.continuuity.net/cdap/2.8.0-SNAPSHOT/en/reference-manual/http-restful-api/http-restful-api-v3/lifecycle.html#start-stop-status-and-runtime-arguments)